frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/TopToolbar/CreateNewWrapper.tsx

```diff
@@ -114,3 +103,13 @@ export function CreateNewWrapper({
     </StudioPopover>
   );
 }
+
+export const extractDataTypeNamesFromAppMetadata = (
```

Comment:
Should this be in a utils file? in validationUtils.ts in XSDUpload fex? And maybe move the file to a utils-folder in the same level as hook below `dataModelling` folder? 🤔